regression/regression_var.sh

```diff
@@ -131,7 +131,7 @@ case $machine in
 
    #  On Hera, there are no scrubbers to remove old contents from stmp* directories.
    #  After completion of regression tests, will remove the regression test subdirecories
-   export clean=".true."
```

Comment:
Was this change intended to be included?  I'm always suspicious when I see changes to the regression drivers.

Reply:
You are correct.  That change was not intended.  I will remove it.